.github/actions/setup-python/action.yml

```diff
@@ -23,5 +23,5 @@ runs:
       if: ${{ inputs.install-dependencies == 'true' }}
       shell: bash
       run: |
-        pip install uv setuptools
+        pip install uv==0.2.37 setuptools==73.0.0
```

Comment:
Curious, why we need to pin here?

Reply:
good catch. I'll revert this, was just testing a few things.

`snyk` was complaining about [a CVE](https://security.snyk.io/vuln/SNYK-PYTHON-SETUPTOOLS-7448482) in `setuptools=68.0.0`. I'm not able to reproduce our environment installing this version though (`engine/requirements.txt` pins `73.0.0` and this GitHub Action, + our Dockerfile, also install `73.0.0` without the pin).. soo I permanently ignored this warning in snyk for now.